Source/WebGPU/WebGPU/WebGPUInternal.h

```diff
@@ -26,6 +26,8 @@
 #pragma once
```

Comment:
this file is malformed; umbrella headers should only include #includes and never any actual symbols